python/pyspark/sql/pandas/group_ops.py

```diff
@@ -413,40 +414,35 @@ def applyInPandas(
         >>> df2 = spark.createDataFrame(
         ...     [(20000101, 1, "x"), (20000101, 2, "y")],
         ...     ("time", "id", "v2"))
-        >>> def asof_join(l, r):
-        ...     return pd.merge_asof(l, r, on="time", by="id")
-        >>> df1.groupby("id").cogroup(df2.groupby("id")).applyInPandas(
-        ...     asof_join, schema="time int, id int, v1 double, v2 string"
-        ... ).show()  # doctest: +SKIP
-        +--------+---+---+---+
-        |    time| id| v1| v2|
-        +--------+---+---+---+
-        |20000101|  1|1.0|  x|
-        |20000102|  1|3.0|  x|
-        |20000101|  2|2.0|  y|
-        |20000102|  2|4.0|  y|
-        +--------+---+---+---+
-
-        Alternatively, the user can define a function that takes three arguments.  In this case,
-        the grouping key(s) will be passed as the first argument and the data will be passed as the
-        second and third arguments.  The grouping key(s) will be passed as a tuple of numpy data
-        types, e.g., `numpy.int32` and `numpy.float64`. The data will still be passed in as two
-        `pandas.DataFrame` containing all columns from the original Spark DataFrames.
-
         >>> def asof_join(k, l, r):
         ...     if k == (1,):
         ...         return pd.merge_asof(l, r, on="time", by="id")
         ...     else:
         ...         return pd.DataFrame(columns=['time', 'id', 'v1', 'v2'])
         >>> df1.groupby("id").cogroup(df2.groupby("id")).applyInPandas(
-        ...     asof_join, "time int, id int, v1 double, v2 string").show()  # doctest: +SKIP
+        ...     asof_join, "time int, id int, v1 double, v2 string").show() # doctest: +SKIP
         +--------+---+---+---+
         |    time| id| v1| v2|
         +--------+---+---+---+
         |20000101|  1|1.0|  x|
         |20000102|  1|3.0|  x|
         +--------+---+---+---+
-
+        >>> df3 = spark.createDataFrame(
+        ...     [(20000101, 1, "x"), (20000101, 2, "y")],
+        ...     ("time", "id", "v3"))
+        >>> def asof_join_multiple(_, *dfs):
```

Comment:
example should mention `key` argument, even if it is not used inside
```suggestion
        >>> def asof_join_multiple(key, *dfs):
```